rust/parquet/src/arrow/arrow_writer.rs

```diff
@@ -189,11 +199,12 @@ fn write_leaves(
         ArrowDataType::Float16 => Err(ParquetError::ArrowError(
             "Float16 arrays not supported".to_string(),
         )),
-        ArrowDataType::FixedSizeList(_, _)
-        | ArrowDataType::FixedSizeBinary(_)
-        | ArrowDataType::Union(_) => Err(ParquetError::NYI(
-            "Attempting to write an Arrow type that is not yet implemented".to_string(),
-        )),
+        ArrowDataType::FixedSizeList(_, _) | ArrowDataType::Union(_) => {
+            Err(ParquetError::NYI(
+                "Attempting to write an Arrow type that is not yet implemented"
+                    .to_string(),
```

Comment:
```suggestion
                format!("Attempting to write an Arrow type {?} to parquet that is not yet implemented", array.data_type())
```

( I realize the error message is not changed by this PR I just saw it when looking)